buildbothammer/src/buildbothammer/__init__.py

```diff
@@ -0,0 +1,381 @@
+import aiohttp
+import asyncio
+import json
+import os
+import logging
+from pathlib import Path
+import subprocess
+from filelock import FileLock
+from github import Github, GithubException
+from github.InputGitTreeElement import InputGitTreeElement
+from github.GithubException import GithubException, UnknownObjectException
+
+# Set up logging
+logging.basicConfig(
+    level=logging.DEBUG, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+logger = logging.getLogger(__name__)
+
+# Buildbot API endpoint
+BUILDBOT_API = "http://buildbot.python.org/api/v2"
+BUILDBOT_URL = "http://buildbot.python.org/#/"
+
+DISCORD_WEBHOOK_URL = os.environ.get("DISCORD_WEBHOOK_URL")
+if not DISCORD_WEBHOOK_URL:
+    logger.warning(
+        "DISCORD_WEBHOOK_URL environment variable is not set. Discord notifications will be disabled."
+    )
+
+# GitHub API token (replace with your own)
+GITHUB_TOKEN = os.environ.get("GITHUB_TOKEN")
```

Comment:
miss-islington is an app now so this won't be super easy. would bedevere's token that we're already using work instead?

Reply:
Yeah we can use that but we need to manually fork the repo with the token. It’s easy to do but right now there is no fork for that user